### PR TITLE
Use database

### PR DESCRIPTION
**Pull request by dneise**

### CODE REVIEW

File: shifthelper_webinterface/__init__.py [hunk 1]
```diff
@@ -57,27 +109,16 @@ def index():
...
-    except:
-        level = alert['level']
-
-    alert['level'] = level
-    alert['acknowledged'] = False
-
-    key = alert['uuid']
-    app.alerts[key] = alert
-
+    add_alert(alert)
```
Comment (dneise) [inline shifthelper_webinterface/__init__.py]: can this fail?

File: shifthelper_webinterface/__init__.py [hunk 2]
```diff
@@ -22,25 +22,77 @@
...
+def retrieve_alerts():
+    comp_date = datetime.utcnow() - timedelta(hours=24)
+    try:
+        alerts = (
+            Alert
+            .select()
+            .order_by(Alert.timestamp.desc())
+            .where(Alert.timestamp > comp_date)
+        )
+    except Alert.DoesNotExist:
```
Comment (dneise) [inline shifthelper_webinterface/__init__.py]: does this ever happen?

**Review by dneise — COMMENTED** [review]
likey

File: shifthelper_webinterface/__init__.py (hunk 2 above)
Comment (maxnoe) [reply shifthelper_webinterface/__init__.py]: Nope, good catch.

File: shifthelper_webinterface/__init__.py (hunk 1 above)
Comment (maxnoe) [reply shifthelper_webinterface/__init__.py]: Probably. I will catch this.

**Pull request merged**